src/superspawn.js

```diff
@@ -17,41 +17,13 @@
     under the License.
 */
 
-var child_process = require('child_process');
+var crossSpawn = require('cross-spawn');
 var fs = require('fs-extra');
-var path = require('path');
 var _ = require('underscore');
 var Q = require('q');
-var which = require('which');
 var events = require('./events');
 var iswin32 = process.platform === 'win32';
 
-// On Windows, spawn() for batch files requires absolute path & having the extension.
-function resolveWindowsExe (cmd) {
-    var winExtensions = ['.exe', '.bat', '.cmd', '.js', '.vbs'];
-    function isValidExe (c) {
-        return winExtensions.includes(path.extname(c)) && fs.existsSync(c);
-    }
-    if (isValidExe(cmd)) {
-        return cmd;
-    }
-    cmd = which.sync(cmd, { nothrow: true }) || cmd;
-    if (!isValidExe(cmd)) {
-        winExtensions.some(function (ext) {
-            if (fs.existsSync(cmd + ext)) {
-                cmd = cmd + ext;
-                return true;
-            }
-        });
-    }
-    return cmd;
-}
-
-function maybeQuote (a) {
-    if (/^[^"].*[ &].*[^"]/.test(a)) return '"' + a + '"';
-    return a;
-}
-
 /**
  * A special implementation for child_process.spawn that handles
```

Comment:
could update that line now as well as `child_process.spawn` is not used any more directly

Reply:
IMHO, the wording is a bit unfortunate to begin with. Do you have a suggestion on how to improve it?

Reply:
> Wrapper for `crossSpawn.spawn` that makes sure `cmd` is executable.

I don't really understand what else it does tbh.

Reply:
Frankly, I would not even mention cross-spawn. I'ts an implementation detail. So maybe we should just change The beginning to

> A `child_process.spawn`-compatible function that handles ...

Reply:
It was the only one I could comment on without actually doing any work :D

Reply:
:rofl: